docs/express.asciidoc

```diff
@@ -0,0 +1,250 @@
+[[express]]
+== Getting started with Express
+
+Getting Elastic APM set up for your Express app is easy,
+and there are various ways you can tweak it to fit to your needs.
+
+Follow the guide below and refer to the <<api,API documentation>> for all the advanced stuff.
+
+[[express-installation]]
+=== Installation
+
+Add the `elastic-apm` module as a dependency to your application:
+
+[source,bash]
+----
+npm install elastic-apm --save
+----
+
+[[express-initialization]]
+=== Initialization
+
+It's important that the agent is started before you require *any* other modules in your Node.js application - i.e. before `express`, `http`, etc.
+
+This means that you should probably require and start it in your applications main file (usually `index.js`, `server.js` or `app.js`).
```

Comment:
Say: "require and start the agent in..." (the pronoun "it" has an unclear antecedent)

Reply:
...in your application's main file (use possessive form)